app/pages/patient/patient.js

```diff
@@ -160,7 +160,17 @@ var Patient = React.createClass({
   },
 
   renderPatientTeam: function() {
-    return this.transferPropsTo(<PatientTeam />);
```

Comment:
All other things being equal, I'd prefer the JSX equivalent of `transferPropsTo` for this, which seems to be the recommendation post-deprecation of `transferPropsTo`: [JSX Spread Attributes](https://facebook.github.io/react/docs/jsx-spread.html). Is there a reason you didn't think this was a good idea here @dotMR?

CC @GordyD because we've been talking about trying to do the JSX spread attributes in more places, potentially.